modules/ROOT/pages/lifecycle.adoc

```diff
@@ -0,0 +1,94 @@
+
+Overwriting an asset version also updates the search index.
+
+The Exchange user interface and APIs change some asset version information for asset versions in any state. This information is not changed by overwriting an asset version in the `development` state. This information includes:
+
+* asset type
+* tags
+* categories
+* custom fields
+* documentation pages
+
+Deleting an asset version in the `development` state is always a hard delete, enabling you to reuse the asset's name, groupId, assetId, and version.
+
+The public portal does not show asset versions in the `development` state.
+
+Anypoint Platform products other than Exchange, such as Anypoint API Designer, Anypoint Flow Designer, or Anypoint Studio, do not consume asset versions in the `development` state.
+
+== Stable State
+
+An asset version in the `stable` state is ready to use. It is not possible to re-publish or overwrite it.
+
+Deleting an asset version in the `stable` state in the first seven days is a hard delete, enabling you to reuse the asset's name, groupId, assetId, and version. Deleting an asset version in the `stable` state after the first seven days is a soft delete, preventing you from reusing the asset's name, groupId, assetId, and version. After a hard delete or a soft delete, an asset cannot be recovered or undeleted.
+
+The Exchange backend and some API responses use the term `published` to mean `stable`.
+
+== Deprecated State
+
+An asset version in the `deprecated` state is similar to an asset version in the `stable` state. The only difference is that the `status` property shows the asset version is deprecated.
+
+== Create an Asset in the Exchange User Interface
+
+. Click *Publish new asset*.
+. Set *Lifecycle state* to the *Development* state or the default *Stable* state.
+. Set other asset information.
+. Click *Publish*.
+
+== State Promotion
+
+These are the three possible state promotions:
+
+. `development` to `stable`
+. `stable` to `deprecated`
+. `deprecated` to `stable`
+
+Promote an asset version's state either in the asset's Exchange portal or by using the Exchange API.
+
+In the asset's portal, click the lifecycle state button that displays the current state, and select the new state of *Development*, *Stable*, or *Deprecated*.
+
+To use the Exchange API, follow the xref:exchange-api.adoc#promote-lifecycle-state[Promote Lifecycle State] example.
+
+== Dependencies
+
+An asset in the `stable` state cannot add an asset in the `development` state as a dependency.
+
+If an asset in the `development` state depends on one or more other assets in the `development` state, Exchange does not prevent promoting that asset to the `stable` state. However, if a dependency in the `development` state is overwritten, the packages of the asset that was promoted to the `stable` state are not regenerated.
```

Comment:
All this text is not valid.

This is not OK,

If an asset in the `development` state depends on one or more other assets in the `development` state, Exchange does not *allow* promoting that asset to the `stable` state, all its dependencies should first be promoted to `stable` first.

Apart from that, if you have an asset in `development` state with dependencies in `development` state and you modify a dependency, the dependent asset will also not be regenerated, it's the user's responsibility to republish the dependent asset once it's dependencies were updated